usaspending_api/etl/management/commands/load_table_from_delta.py

```diff
@@ -215,196 +367,180 @@ def handle(self, *args, **options):
+            self.logger.info(f"Creating {qualified_temp_table}")
+            cursor.execute(create_temp_sql)
+            self.logger.info(f"{qualified_temp_table} created.")
+
+            if is_postgres_table_partitioned and partitions_sql:
+                for create_partition in partitions_sql:
+                    self.logger.info(f"Creating partition of {qualified_temp_table} with SQL:\n{create_partition}")
+                    cursor.execute(create_partition)
+                    self.logger.info("Partition created.")
+
+            # If there are vectors, add the triggers that will populate them based on other calls
+            # NOTE: Undetermined whether tsvector triggers can be applied on partitioned tables,
+            #       at the top-level virtual/partitioned table (versus having to apply on each partition)
+            tsvectors = self.table_spec.get("tsvectors") or {}
+            for tsvector_name, derived_from_cols in tsvectors.items():
+                self.logger.info(
+                    f"To prevent any confusion or duplicates, dropping the trigger"
+                    f" tsvector_update_{tsvector_name} if it exists before potentially recreating it."
+                )
+                cursor.execute(f"DROP TRIGGER IF EXISTS tsvector_update_{tsvector_name} ON {qualified_temp_table}")
 
-        # Make sure that the column order defined in the Delta table schema matches
-        # that of the Spark dataframe used to pull from the Postgres table. While not
-        # always needed, this should help to prevent any future mismatch between the two.
-        if column_names:
-            df = df.select(column_names)
+                self.logger.info(
+                    f"Adding tsvector trigger for column {tsvector_name}"
+                    f" based on the following columns: {derived_from_cols}"
+                )
+                derived_from_cols_str = ", ".join(derived_from_cols)
+                tsvector_trigger_sql = f"""
+                    CREATE TRIGGER tsvector_update_{tsvector_name} BEFORE INSERT OR UPDATE
+                    ON {qualified_temp_table} FOR EACH ROW EXECUTE PROCEDURE
+                    tsvector_update_trigger({tsvector_name}, '{DEFAULT_TEXT_SEARCH_CONFIG}',
+                                            {derived_from_cols_str})
+                """
+                cursor.execute(tsvector_trigger_sql)
+                self.logger.info(f"tsvector trigger for column {tsvector_name} added.")
+
+        return qualified_temp_table
 
-        # If we're working off an existing table, truncate before loading in all the data
-        if not make_new_table:
-            self.logger.info(f"Truncating existing table {temp_table}")
-            with db.connection.cursor() as cursor:
-                cursor.execute(f"TRUNCATE {temp_table}")
-                self.logger.info(f"{temp_table} truncated.")
+    def _set_sequence_value(self, seq_name: str, val: Optional[int] = None) -> int:
+        """
```

Comment:
This pydoc doesn't follow the existing pattern in the returns part of the string. For example, here is a method that shows (what I've come to understand as) the standard pydoc. Notice how returns is preceded by a "Returns:"

```
    def country_results(self, loc_lookup: str) -> List[dict]:
        """Find subaward results for countries

        Args:
            loc_lookup (String): Name of the field on the SubawardSearch model to use to find subawards.

        Returns:
            List[dict]: List of subaward results by country
        """
```

Reply:
Fixed.